samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -52,6 +54,18 @@ public static async Task Main(string[] args)
         // Add SignalR as the real time relay service
         builder.Services.AddSignalR();
 
+        // Add TesseractEngine if we find the language file in the specified location.
+        var tesseractFilePath = builder.Configuration["Tesseract:FilePath"];
+        var tesseractLanguage = builder.Configuration["Tesseract:Language"];
+        if (!string.IsNullOrEmpty(tesseractFilePath) && !string.IsNullOrEmpty(tesseractLanguage) && System.IO.File.Exists(System.IO.Path.Combine(tesseractFilePath, $"{tesseractLanguage}.traineddata")))
```

Comment:
you shouldn't need to do this since `Tesseract.FilePath` should be an exact path to the data file, similar to `ChatStore.Filesystem.FilePath`.

Reply:
This was a defensive measure to avoid runtime errors setting the TesseractEngine on start up when the file was not found or if the developer did not want to enable OCR support.  

I've rewrote this now to do what was originally asked about potentially supporting Cognitive Services for OCR (as well as potentially other OCR options) by adding in a higher level "OcrSupport" part in the config and then allowing a "none" type.